lib/Serverless.js

```diff
@@ -70,6 +71,12 @@ class Serverless {
       return this.pluginManager.run(this.processedInput.commands);
     }
 
+    // check if tracking is enabled (and track if it's enabled)
+    const serverlessPath = this.config.serverlessPath;
```

Comment:
@eahefnawy could you maybe take a look at this when you have a sec? That would be really great!

Should be a "simple" try catch which checks if the user has permissions (maybe needed in the `toggleTracking` plugin method and in the `lib/Serverless.js` file where the check if the file exists happens).